receiver/sqlserverreceiver/README.md

```diff
@@ -36,6 +38,14 @@ Windows-specific options:
 - `computer_name` (optional): The computer name identifies the SQL Server name or IP address of the computer being monitored.
   If specified, `instance_name` is also required to be defined. This option is ignored in non-Windows environments.
 
+Top-Query collection specific options (only useful when top-query collection are enabled):
+- `lookback_time` (optional, example = `60`, default = `2 * collection_interval`): The time window (in second) in which to query for top queries.
+  - Queries that were finished execution outside the lookback window are not included in the collection. Increasing the lookback window (in seconds) will be useful for capturing long-running queries.
+- `max_query_sample_count` (optional, example = `5000`, default = `1000`): The maximum number of records to fetch in a single run.
+- `top_query_count`: (optional, example = `100`, default = `200`): The maximum number of active queries to report (to the next consumer) in a single run.
+- `enable_top_query_collection`: (optional, default = `false`): Enable collection of top queries.
+  - e.g. `sqlserver` receiver will fetch 1000 (value: `max_query_sample_count`) queries from database and report the top 200 (value: `top_query_count`) which used the most CPU time.
```

Comment:
Should we change this doc corresponding to the actual config?

Reply:
sorry i don't quite follow, the document shows the default config. can you elaborate about the `actual config`? thanks

Reply:
This is the actual config looks like:

```yaml
        top_query_collection:
          enabled: true
          lookback_time: 60
          max_query_sample_count: 1000
          top_query_count: 200
```

There is no field called `enable_top_query_collection`

Reply:
updated